File: src/main/java/com/findcheeseheads/backend/model/Report.java
package com.findcheeseheads.backend.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;

@JsonSerialize
public class Report {

    @JsonProperty
    public Integer venueId;

    @JsonProperty
    public String reason;

    @JsonProperty
    public String other;

    @JsonProperty
    public int count;
}

File: src/main/angular/app/model/report.ts
export class Report {
  public type: string = null;
  public count: number = undefined;
}

export class ReportMessage {
  public venueId: number;
  public success = false;
  public message: string;
}

export class ReportSubmission {
  public venueId: number = null;
  public reason: string = null;
  public other: string = null;
}
  
